src/app/simple-dvt-deploy.e2e-chain.ts

```diff
@@ -100,7 +100,7 @@ describe('Simple DVT deploy', () => {
     stakingRouterService = moduleRef.get(StakingRouterService);
 
     jest.spyOn(configService, 'get').mockImplementation((path: any) => {
-      if (path === 'LIDO_LOCATOR_ADDRESS') {
+      if (path === 'LIDO_LOCATOR_DEVNET_ADDRESS') {
```

Comment:
how does it work for holesky? 

will it always use in test devnet locator address?

Reply:
In another review we discussed that it is better to call this variable LIDO_LOCATOR_DEVNET_ADDRESS since it is only used in Devnet